documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot-nosql.adoc

```diff
@@ -83,6 +84,13 @@ The array lists regular expressions which match {data-collection}s by their full
 |An optional `type` component of the `data` field of a signal that specifies the type of snapshot operation to run. +
 Currently supports the `incremental` and `blocking` types. +
 If you do not specify a value, the connector runs an incremental snapshot.
+
+|6
+|`additional-conditions`
+| An optional array that specifies a set of additional conditions that the connector evaluates to determine the subset of records to include in a snapshot. +
+Each additional condition is an object with `data-collection` and `filter` properties. You can specify different filters for each data collection. +
+* The `data-collection` property is the fully-qualified name of the data collection for which the filter will be applied.
```

Comment:
```suggestion
Each additional condition is an object with `data-collection` and `filter` properties. You can specify different filters for each data collection. +
* The `data-collection` property is the fully-qualified name of the data collection for which the filter will be applied.
```
I think that we still have a problem in this passage. L91 states that _"Each additional condition is an object with `data-collection` and `filter` properties"_.  I think that sentence is intended to introduce the unordered list that follows, but the list contains only the single entry, `data-collection`, and no entry for the `filter` property. 

If it is correct that we don't also have a `filter` property, the reference to it should be removed on L91. Or if you set the filter elsewhere, provide a reference to where and how that's done. 

If there's no `filter` property L91 should omit the mention it, and we should remove the asterisk/bullet on L92, letting it just stand as a plain sentence. Currently, we signal an unordered list, and provide an introduction that suggests the list will include two entries, but then we only provide the first item. 
Does that make sense? Let me know if you want to discuss further.

Reply:
`data-collection`:: The fully-qualified name of the data collection for which the filter will be applied. 
`filter`:: Specifies the column values that must be present in a data collection record for the snapshot to include it, for example, `"color='blue'"`.